tt_eager/tt_dnn/op_library/reduce/reduce_op.cpp

```diff
@@ -190,7 +190,8 @@ Tensor mean_hw(const Tensor& input_tensor, const MemoryConfig& output_mem_config
     return mean(input_tensor,2,output_mem_config);
 }
 Tensor global_mean(const Tensor& input_tensor, const MemoryConfig& output_mem_config) {
-    return mean(input_tensor,4,output_mem_config);
+    float inv_volume = 1.0f/input_tensor.volume();
+    return mul_unary_sfpu( inv_volume, global_sum(input_tensor,output_mem_config), output_mem_config);
```

Comment:
Is there a reason we need to normalize as a separate op since reduce can take a scaler arg to do this?
Or is accuracy too low since it is prescaling in reduce compared to post scaling done here?

Reply:
When you say `this is the only configuration of mean that works well` do you mean that accuracy is low if you use the scaler arg of reduce?